modules/calib/include/opencv2/calib.hpp

```diff
@@ -1122,6 +1122,40 @@ CV_EXPORTS_W double stereoCalibrate( InputArrayOfArrays objectPoints,
                                      OutputArray perViewErrors, int flags = CALIB_FIX_INTRINSIC,
                                      TermCriteria criteria = TermCriteria(TermCriteria::COUNT+TermCriteria::EPS, 30, 1e-6) );
 
+/** @brief Estimates intrinsics and extrinsics (camera pose) for multi-camera system.
+Multiview calibraton.
```

Comment:
Malformed documentation:

> One could use the [\brief](https://www.doxygen.nl/manual/commands.html#cmdbrief) command with one of the above comment blocks. This command ends at the end of a paragraph, so the detailed description follows **after an empty line**.

Ref: https://www.doxygen.nl/manual/docblocks.html

The same as https://github.com/opencv/opencv/pull/23098#discussion_r1105320043 (1 week and the same again)